ddl/sanity_check.go

```diff
@@ -97,7 +97,7 @@ func expectedDeleteRangeCnt(ctx delRangeCntCtx, job *model.Job) (int, error) {
 		if err := job.DecodeArgs(&startKey, &physicalTableIDs, &ruleIDs); err != nil {
 			return 0, errors.Trace(err)
 		}
-		return mathutil.Max(len(physicalTableIDs), 1), nil
+		return mathutil.Max(len(physicalTableIDs)+1, 1), nil
```

Comment:
Why `Max(x+1, 1)` if x >= 0?

Reply:
For drop table ddl the [t{tableID}_, t{tableID+1}) range is deleted in `finishDDLJob`, as logical table may contain global index.

Reply:
I don't think the `max` function is needed anymore here. Since `len(xx)` is always >= 0, the return value of the `max` function is always equal to `len(physicalTableIDs)+1`.
```suggestion
		return len(physicalTableIDs)+1, nil
```

Reply:
@Defined2014  resolved, thanks.